kolibri/plugins/management/assets/src/views/top-nav/index.vue

```diff
@@ -2,19 +2,34 @@
 
   <div class="top">
     <div class="links">
-      <router-link :to="classesLink" :class="{active: classesActive}" @click.native="blur">
+      <router-link
+        v-if="!isSuperuser"
+        :to="classesLink"
+        :class="{active: classesActive}"
+        @click.native="blur"
+      >
         {{$tr('classes')}}
       </router-link>
-      <router-link :to="usersLink" :class="{active: usersActive}" @click.native="blur">
+      <router-link
+        v-if="!isSuperuser"
```

Comment:
I think @indirectlylit's motivation here was to disincentivize the use of the Device Owner account in general, because it causes issues.